### PR TITLE
Do the thing where I clean up var/level

### PR DESCRIPTION
**Pull request by Laboredih123**
[INTERNAL][CLEAN]
## About the PR 
DMDocs var/level and macros its values.

## Why's this needed? 
I like to know what the numbers do.

### CODE REVIEW

File: code/modules/mechanics/MechanicMadness.dm [hunk 1]
```diff
@@ -482,11 +482,11 @@
 		return ..()
 
 	pick_up_by(var/mob/M)
-		if(level != 1) return ..()
+		if(level == UNDERFLOOR) return ..()
```
Comment (TobleroneSwordfish) [inline code/modules/mechanics/MechanicMadness.dm:485]: this is backwards I think

File: code/obj.dm [hunk 2]
```diff
@@ -356,6 +356,7 @@
 /obj/proc/alter_health()
 	return 1
 
+/// Whether or not to hide something based on the value of h.
```
Comment (TobleroneSwordfish) [inline code/obj.dm:359]: h

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Comment by BatElite** [comment]
The connector port has a level of 0 and it seems to be the only thing that does so. Might as well change that to overfloor.

(god, imagine underfloor connector ports. evil.)

File: code/modules/mechanics/MechanicMadness.dm (hunk 1 above)
Comment (Laboredih123) [reply code/modules/mechanics/MechanicMadness.dm:485]: feck

File: code/obj.dm (hunk 2 above)
Comment (Laboredih123) [reply code/obj.dm:359]: h

**Pull request merged**